### ISSUE TITLE
No documentation on how to use Reciter and the Reciter Pubmed retrieval tool

### ISSUE THREAD

**Issue by jcabrerazuniga** [wcmc-its/ReCiter#500]
I have running Reciter and the Reciter Pubmed Retrieval tool. Yet, I do not see how to test their integration. Could your documentation be enhanced to provide some example where Reciter is forced to use the Reciter pubmed retrieval tool?

Thanks

**Comment by paulalbert1** [comment]
Hey @jcabrerazuniga - The PubMed Retrieval Tool is required, which is implied [here](https://github.com/wcmc-its/ReCiter#related-code-repositories). Are you saying that's ambiguous from the readme? If I said somewhere that the Retrieval Tool is required, would that address your concern?

For your second question, how do you test the integration between the two appsl? You run a Feature Generator for a person who has an identity record.

Is this helpful?

**Comment by jcabrerazuniga** [comment]
I know the retrieval tool is required and that's the reason I installed it. Can you please tell me how can I retrieve all persons who have an identity record? I also could not find the default values for the Reciter's database that I will like to query. 

Thanks

**Comment by paulalbert1** [comment]
There are several options for retrieving data from DynamoDB.

1. You can retrieve all users from DynamoDB using the [Find All Identity API](https://github.com/wcmc-its/ReCiter/wiki/Using-the-APIs#reciterfindallidentity). 
2. You can download some of the data from the DynamoDB / AWS interface but I think that's only the personIdentifier.
3. There are a couple of third party tools for querying DynamoDB including [this](https://github.com/bchew/dynamodump) and [this](https://github.com/edasque/DynamoDBtoCSV).
4. Another option is to use the MySQL database and a set of Python scripts available [here](https://github.com/wcmc-its/reciter-machineLearning-Analysis/). But we're in the process of refactoring these, so that's not as well documented. Generally speaking, the MySQL database gives you (or will give you) the most options. 

I'm not sure which Reciter database you're referring to: the DynamoDB one or the MySQL one.

If you want to sync up over Zoom, send me a couple times next week.
-- Paul, paa2013@med.cornell.edu

**Comment by jcabrerazuniga** [comment]
I noticed while compiling Reciter using: 

aws.dynamoDb.local=true 
aws.dynamoDb.local.port=8000
aws.dynamoDb.local.region=us-central-1
**aws.dynamoDb.local.accesskey=dummy
aws.dynamoDb.local.secretkey=dummy**
aws.dynamoDb.local.dbpath=/opt/ReCiter/dynamodb/

that it is installs a local copy of DynamoDB  which is possible according to https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html. 

I think I just found while I was looking for... the default username and password are "dummy". But another question arises: Do I need to re-compile Reciter if I decide to change those passwords?

Thanks
